cmd/panicbot/main.go

```diff
@@ -2,25 +2,141 @@ package main
 
 import (
 	"fmt"
+	"os"
 
 	"github.com/bwmarrin/discordgo"
-	"github.com/k0kubun/pp/v3"
-	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
 	"github.com/twilio/twilio-go"
+	"sigs.k8s.io/yaml"
 )
 
+// type Boot interface {
+// 	configChanged() error
+// 	configureLogger()
+// 	watchFile(filePath string) error
+// }
 type Config struct {
-	// TODO: Put config here.
+	DiscordBotToken string
+	GuildID         string
+	RoleID          string
 }
 
 type Container struct {
 	Config           Config
-	Logger           *logrus.Logger
-	Session          *discordgo.Session
+	Logger           *log.Logger
+	Discord          *discordgo.Session
 	TwilioRestClient *twilio.RestClient
 }
 
+// var _ Boot = (*Container)(nil)
+
 func main() {
-	fmt.Println("Hello Worlds")
-	pp.Println("string")
+	c := new(Container)
+	c.configureLogger()
+	err := c.configChanged(true)
+	if err != nil {
+		c.Logger.Fatalf("failed to load config: %s", err.Error())
+	}
+	err = c.startReloadRolesTimer()
+	if err != nil {
+		c.Logger.Fatalf("failed to start timer to check for update roles : %s", err.Error())
+	}
+	err = c.registerSlashCommands()
+	if err != nil {
+		c.Logger.Fatalf("failed to register slash commands : %s", err.Error())
+	}
+	// err = c.watchFile("./config.yml")
+	// if err != nil {
+	// 	c.Logger.Fatalf("failed to watch configuration file: %s", err.Error())
+	// }
+}
+
+func (c *Container) configChanged(load bool) error {
+	yfile, err := os.ReadFile("./config.yml")
```

Comment:
lets grab this from an environment variable or from ./config.yml if the var is ""